ios/src/TGGeoPolygon.mm

```diff
@@ -0,0 +1,80 @@
+//
+//  TGGeoPolygon.mm
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#import "TGGeoPolygon.h"
+
+#include <vector>
+
+@interface TGGeoPolygon () {
+    std::vector<TGGeoPoint> coordinates;
+    std::vector<int> rings;
```

Comment:
Weeeird. How would one create a "dynamic array" of integers in Objective-C then?